eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,188 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    # Download package from nuget, if no package found in nuget, then search devops public feeds if set.
+    Write-Host "Downloading source: '$customPackageSource'. If it is empty, we default to use nuget gallery."
```

Comment:
It is still worth having the package name and version for diagnostic purposes.

Reply:
Actually, we have the $package $version information logged out. No need to log here it is a redundant information. 
https://github.com/Azure/azure-sdk-for-net/pull/32062/files#diff-4b590925689e1df66d5c28ddc67d15ded4db6d686eb7c7878e0c8238feb57e25R45